src/lib/DateInput.svelte

```diff
@@ -94,6 +94,9 @@
   /** Wait with updating the date until a date is selected */
   export let browseWithoutSelecting = false
 
+  /** Allow Users to select time */
+  export let timePicker: boolean = false
```

Comment:
```suggestion
  export let timePrecision: null | 'minute' = null
```

Would like it typed like this so `second` and `millisecond` can be supported

Reply:
Okay

Reply:
Hello  @probablykasper , sorry for the delay.

> Would like it typed like this so `second` and `millisecond` can be supported

Have pushed the other requested changes, however on a closer look, it seems you want me to use timePrecision instead of timePicker.
Or, would you like another API parameter that allows user to set the timePrecision?

Now, even if we do take  timePrecision to allow seconds and milliseconds, how would that work from the UI?

Reply:
No problem. It would be instead of the timePicker prop yes, so if timePrecision == null, it doesn't show up.

The UI would be similar to https://blueprintjs.com/docs/#datetime2/date-picker3

Reply:
thanks for this